src/window/settings.rs

```diff
@@ -35,6 +37,8 @@ impl Default for WindowSettings {
             touch_drag_timeout: 0.17,
             background_color: "".to_string(),
             confirm_quit: true,
+            font_edging: "AntiAlias".to_string(),
+            font_hinting: "Full".to_string()
```

Comment:
All other settings are in lowercase, except these two. I'd prefer consistency here and by that, also adjust the docs, but your choice.

Reply:
Done, all of them are lowercase now.